### PR TITLE
Add managed-scripts configs

### PR DESCRIPTION
**Pull request by wanghaoran1988**

### CODE REVIEW

File: core-services/prow/02_config/openshift/managed-scripts/_pluginconfig.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  - openshift/managed-scripts
+  require_self_approval: false
+lgtm:
+- repos:
+  - openshift/managed-scripts
+  review_acts_as_lgtm: true
+plugins:
+  openshift/managed-scripts:
+    plugins:
+    - approve
```
Comment (petr-muller) [inline core-services/prow/02_config/openshift/managed-scripts/_pluginconfig.yaml:13]: In order for `lgtm` and `approve` to make sense, you also need to configure Tide, their consumer (see other repositories `_prowconfig.yaml` configuration).

Comment (wanghaoran1988) [reply core-services/prow/02_config/openshift/managed-scripts/_pluginconfig.yaml:13]: @petr-muller Could you check again?

**Review by petr-muller — APPROVED** [review]
LGTM

**Pull request merged**